### PR TITLE
Add Payments APIs overviews

### PR DESCRIPTION
**Pull request by henriquessb**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

### CODE REVIEW

File: VTEX - Antifraud Provider API.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "openapi": "3.0.0",
     "info": {
         "title": "Antifraud Provider",
-        "description": "Endpoints that deal with to Antifraud Provider operations",
+        "description": ">ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/vtex-rest-api/docs/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe Antifraud Provider Protocol is a set of definitions to help you integrate your antifraud service API into VTEX platform.\r\n\r\nTo achieve this, you need to implement a web API (REST) following the specifications described in this documentation.\r\n\r\nTo learn more about the Antifraud Provider Protocol, check our [Help Center article](https://help.vtex.com/tutorial/antifraud-provider--4aZtmdpgFikcsQomWyqAOq).",
```
Comment (julia-rabello) [inline VTEX - Antifraud Provider API.json:5]:
```suggestion
        "description": ">ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/docs/guides/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe Antifraud Provider Protocol is a set of definitions to help you integrate your antifraud service API into VTEX platform.\r\n\r\nTo achieve this, you need to implement a web API (REST) following the specifications described in this documentation.\r\n\r\nTo learn more about the Antifraud Provider Protocol, check our [Help Center article](https://help.vtex.com/tutorial/antifraud-provider--4aZtmdpgFikcsQomWyqAOq).",
```

File: VTEX - Customer Credit API.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+        "description": "With Customer Credit your store can enable **credit payments** through the checkout. You can also control **invoices** and the **credit limits** of your clients.\r\n\r\nLearn more about Customer Credit in our [Help Center article](https://help.vtex.com/en/tracks/customer-credit-getting-started).\r\n\r\nAll requests need the **authorization header**.\r\n\r\nAdditionally, you can find more information on installment payments for an order in the `customData`  field found in the [Get Order](https://developers.vtex.com/reference/orders#getorder) endpoint of the Orders API. This includes the number of installments, amount and due dates.\r\n\r\nThis API allows two kinds of authorization:\r\n1. Authorization header containing the VTEX ID authentication token.\r\n2. VTEX Appkey and Apptoken headers.",
```
Comment (julia-rabello) [inline VTEX - Customer Credit API.json:5]:
```suggestion
        "description": "With Customer Credit your store can enable **credit payments** through the checkout. You can also control **invoices** and the **credit limits** of your clients.\r\n\r\nLearn more about Customer Credit in our [Help Center article](https://help.vtex.com/en/tracks/customer-credit-getting-started--1hCRg21lXYy2seOKgqQ2CC/36grlQ69NK6OCuioeekyCs).\r\n\r\nAll requests need the **authorization header**.\r\n\r\nAdditionally, you can find more information on installment payments for an order in the `customData`  field found in the [Get Order](https://developers.vtex.com/docs/api-reference/orders-api#get-/api/oms/pvt/orders/-orderId-) endpoint of the Orders API. This includes the number of installments, amount and due dates.\r\n\r\nThis API allows two kinds of authorization:\r\n1. Authorization header containing the VTEX ID authentication token.\r\n2. VTEX Appkey and Apptoken headers.",
```

File: VTEX - Giftcard Provider Protocol.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
...Payments onboarding guide](https://developers.vtex.com/vtex-rest-api/docs/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe **Gift Card Provider Protocol** is a set of definitions to help you integrate your Gift Card API into VTEX platform. \r\n\r\nTo achieve this, you need to implement a web API (REST) following the specifications from this documentation and configure your provider in your store using [Gift Card Hub API](https://developers.vtex.com/docs/api-reference/giftcard-hub-api#overview).\r\n\r\n## Common parameters in the documentation\r\n\r\n<table>\r\n  <tr>\r\n    <td><code>{{providerApiEndpoint}}</code></td>\r\n    <td>Provider's endpoint for the implementation (mandatory)</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-PROVIDER-API-AppKey}}</code></td>\r\n    <td>The AppKey configured by the merchant (optional)</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-PROVIDER-API-AppToken}}</code></td>\r\n    <td>The AppToken configured by the merchant (optional)</td>\r\n  </tr>\r\n</table>",
```
Comment (julia-rabello) [inline VTEX - Giftcard Provider Protocol.json:5]:
```suggestion
		"description": ">ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/docs/guides/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe **Gift Card Provider Protocol** is a set of definitions to help you integrate your Gift Card API into VTEX platform. \r\n\r\nTo achieve this, you need to implement a web API (REST) following the specifications from this documentation and configure your provider in your store using [Gift Card Hub API](https://developers.vtex.com/docs/api-reference/giftcard-hub-api#overview).\r\n\r\n## Common parameters in the documentation\r\n\r\n<table>\r\n  <tr>\r\n    <td><code>{{providerApiEndpoint}}</code></td>\r\n    <td>Provider's endpoint for the implementation (mandatory)</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-PROVIDER-API-AppKey}}</code></td>\r\n    <td>The AppKey configured by the merchant (optional)</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-PROVIDER-API-AppToken}}</code></td>\r\n    <td>The AppToken configured by the merchant (optional)</td>\r\n  </tr>\r\n</table>",
```

File: VTEX - Payment Provider Protocol.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
...egration-pci-dss-compliance) or use [Secure Proxy](https://developers.vtex.com/vtex-rest-api/docs/payments-integration-secure-proxy) to accept branded cards like Visa, Mastercard, Amex, etc.\r\n\r\n## Testing\r\n\r\nYou can perform tests on your payment using our [VTEX - Payment Provider Test Suite](https://apps.vtex.com/vtex-payment-provider-test-suite/p).\r\n\r\n>\u2139 An extra header `X-VTEX-API-Is-TestSuite` = `true` will be sent for every test suite request, to help you easily identify it and mock any required scenarios.\r\n\r\nFor more details about testing a Payment Provider, check our [Payment Provider Homologation](https://developers.vtex.com/docs/guides/payments-integration-payment-provider-homologation) article.\r\n\r\n## Common parameters in the documentation\r\n\r\n<table>\r\n  <tr>\r\n    <td><code>{{providerApiEndpoint}}</code></td>\r\n    <td>Provider's endpoint for the implementation</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-VTEX-API-AppKey}}</code></td>\r\n    <td>The AppKey configured by the merchant</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-VTEX-API-AppToken}}</code></td>\r\n    <td>The AppToken configured by the merchant</td>\r\n  </tr>\r\n</table>",
```
Comment (julia-rabello) [inline VTEX - Payment Provider Protocol.json:5]:
```suggestion
        "description": ">ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/docs/guides/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe Payment Provider Protocol is a set of definitions to help you integrate your payment processing API into the VTEX platform, including sub-acquirers, acquirers, gateways, and custom payment services. \r\n\r\nTo achieve this, you must implement a web API (REST) following the specifications described in this documentation.\r\n\r\nTo learn more about the Payment Provider Protocol, check our [Help Center article](https://help.vtex.com/en/tutorial/payment-provider-protocol--RdsT2spdq80MMwwOeEq0m).\r\n\r\n## Endpoint requirements\r\n\r\n  * Must use a standard subdomain/domain name, not an IP address.\r\n  * Must be served over HTTPS on port 443 with TLS 1.2 support.\r\n  * Must respond in less than 5 seconds when running the tests.\r\n  * Must respond in less than 20 seconds when in production.\r\n  * Must be [PCI-DSS compliant](https://developers.vtex.com/docs/guides/payments-integration-pci-dss-compliance) or use [Secure Proxy](https://developers.vtex.com/docs/guides/payments-integration-secure-proxy) to accept branded cards like Visa, Mastercard, Amex, etc.\r\n\r\n## Testing\r\n\r\nYou can perform tests on your payment using our [VTEX - Payment Provider Test Suite](https://apps.vtex.com/vtex-payment-provider-test-suite/p).\r\n\r\n>\u2139 An extra header `X-VTEX-API-Is-TestSuite` = `true` will be sent for every test suite request, to help you easily identify it and mock any required scenarios.\r\n\r\nFor more details about testing a Payment Provider, check our [Payment Provider Homologation](https://developers.vtex.com/docs/guides/payments-integration-payment-provider-homologation) article.\r\n\r\n## Common parameters in the documentation\r\n\r\n<table>\r\n  <tr>\r\n    <td><code>{{providerApiEndpoint}}</code></td>\r\n    <td>Provider's endpoint for the implementation</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-VTEX-API-AppKey}}</code></td>\r\n    <td>The AppKey configured by the merchant</td>\r\n  </tr>\r\n  <tr>\r\n    <td><code>{{X-VTEX-API-AppToken}}</code></td>\r\n    <td>The AppToken configured by the merchant</td>\r\n  </tr>\r\n</table>",
```

File: VTEX - Payments Gateway API.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
     "openapi": "3.0.0",
     "info": {
         "title": "Payments Gateway API",
-        "description": "Here you will find REST Calls available for get transaction details and payment process status.\n",
+        "description": ">ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/vtex-rest-api/docs/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe Payments Gateway API allows you to get payment data and process your store's transactions.",
```
Comment (julia-rabello) [inline VTEX - Payments Gateway API.json:5]:
```suggestion
        "description": ">ℹ️ Onboarding guide\r\n>\r\n> Check the new [Payments onboarding guide](https://developers.vtex.com/docs/guides/payments-overview). We created this guide to improve the onboarding experience for developers at VTEX. It assembles all documentation on our Developer Portal about Payments and is organized by focusing on the developer's journey.\r\n\r\nThe Payments Gateway API allows you to get payment data and process your store's transactions.",
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: VTEX - Giftcard Provider Protocol.json (hunk 3 above)
Comment (henriquessb) [reply VTEX - Giftcard Provider Protocol.json:5]: Houve alguma mudança nessa sugestão? Eu não vi diferença.

File: VTEX - Payment Provider Protocol.json (hunk 4 above)
Comment (henriquessb) [reply VTEX - Payment Provider Protocol.json:5]: Houve alguma mudança nessa sugestão? Eu não vi diferença.

File: VTEX - Giftcard Provider Protocol.json (hunk 3 above)
Comment (julia-rabello) [reply VTEX - Giftcard Provider Protocol.json:5]: Alterei a URL para [https://developers.vtex.com/docs/guides/payments-overview](https://developers.vtex.com/docs/guides/payments-overview)

File: VTEX - Payment Provider Protocol.json (hunk 4 above)
Comment (julia-rabello) [reply VTEX - Payment Provider Protocol.json:5]: Alterei algumas URLs para /docs/guides/ e incluí a ID na do Help

**Review by julia-rabello — APPROVED** [review]
LGTM

**Pull request merged**